src/tests/ftest/util/general_utils.py

```diff
@@ -1207,3 +1207,31 @@ def check_ssh(log, hosts, cmd_timeout=60, verbose=True):
     """
     result = run_remote(log, hosts, "uname", timeout=cmd_timeout, verbose=verbose)
     return result.passed
+
+
+def find_library(name):
+    """Find a library by a given name.
+
+    In order of preference, searches in
+      LD_LIBRARY_PATH
+      MPI_LIB
```

Comment:
MPI_LIB is not set until after `module load mpi`, which is where the library is in CI